spec/factories/students.rb

```diff
@@ -87,6 +88,7 @@
     guardian_two_phone_type    { EnumConstants::PHONE_TYPES.sample }
     guardian_two_relationship  { EnumConstants::GUARDIAN_RELATIONSHIPS.sample }
     health_conditions          { EnumConstants::BOOLEANS.sample }
+    health_conditions_other    { 'NONE' }
```

Comment:
As well, none of the model annotations had a field `health_conditions_other` - did you forget to run `annotate` after creating that field?